lib/registration.pm

```diff
@@ -191,6 +193,42 @@ sub register_product {
     assert_script_run 'SUSEConnect -r ' . get_required_var('SCC_REGCODE');
 }
 
+=head2 register_product_proxyscc
+ 
+    register_product_proxyscc();
+
+Wrapper for SUSEConnect --url <scc_proxy_url> -r <regcode>.
+Requires SCC_URL and SCC_REGCODE variable.
+=cut
+sub register_product_proxyscc {
+    assert_script_run 'SUSEConnect --url ' . get_required_var('SCC_URL') . ' -r ' . get_required_var('SCC_REGCODE');
+}
+
+sub register_addons_text {
+    my ($addonlist) = @_;
+    $addonlist //= get_var('SCC_ADDONS');
+    my @addons = grep { defined $_ && $_ } split(/,/, $addonlist);
+    foreach my $addon (@addons) {
+        my $name = get_addon_fullname($addon);
+        if ($name =~ /module/) {
+            my @ver = split(/\./, scc_version());
+            add_suseconnect_product($name, $ver[0]);
+        }
+        elsif ($name =~ /live/) {
```

Comment:
I think this ticket is focus on the workaround to register modules after media migration to SLE12SP5, we shouldn't enable extensions.

Reply:
Need discuss

Reply:
Keep this registration as discussed with Sunny